sovrin_client/test/agent/base_agent.py

```diff
@@ -0,0 +1,183 @@
+import os
+
+from ioflo.base.consoling import Console
+
+from plenum.common.log import Logger, getlogger
+from sovrin_client.agent.agent import runBootstrap
+
+from sovrin_client.test.agent.test_walleted_agent import TestWalletedAgent
+
+from plenum.common.txn import NAME, VERSION
+
+from anoncreds.protocol.types import ID
+from sovrin_client.agent.exception import NonceNotFound
+from sovrin_client.client.client import Client
+from sovrin_client.client.wallet.wallet import Wallet
+from sovrin_client.test.helper import primes
+from sovrin_common.config import agentLoggingLevel
+from sovrin_common.config_util import getConfig
+
+
+class BaseAgent(TestWalletedAgent):
```

Comment:
I was thinking that, but wasn't sure about the purpose of both (like the base agent was created separately), so for now I have kept it separate. May be we do that as a next step.